inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -70,36 +114,37 @@ std::pair<bool, std::string> compare_functions(
 
     auto f1_results = f1->get_results();
     auto f2_results = f2->get_results();
-    auto compare_nodes_by_name = [](const std::shared_ptr<ngraph::Node> & l, const std::shared_ptr<ngraph::Node> & r)
-            { return l->get_friendly_name() < r->get_friendly_name(); };
-    std::sort(f1_results.begin(), f1_results.end(), compare_nodes_by_name);
-    std::sort(f2_results.begin(), f2_results.end(), compare_nodes_by_name);
+
+    std::sort(f1_results.begin(), f1_results.end(), less_by_name);
+    std::sort(f2_results.begin(), f2_results.end(), less_by_name);
+
     if (f1_results.size() != f2_results.size()) {
-        return { false, "Number of results is different: " + std::to_string(f1_results.size()) + " and " + std::to_string(f2_results.size()) };
+        return error(
+            "Number of results is different: " + to_str(f1_results.size()) + " and " + to_str(f2_results.size()));
     }
 
     const auto& f1_sinks = f1->get_sinks();
     const auto& f2_sinks = f2->get_sinks();
     if (f1_sinks.size() != f2_sinks.size()) {
-        return { false, "Number of sinks is different: " + std::to_string(f1_sinks.size()) + " and " + std::to_string(f2_sinks.size()) };
+        return error(
+            "Number of sinks is different: " + to_str(f1_sinks.size()) + " and " + to_str(f2_sinks.size()));
     }
 
-    auto typeInfoToStr = [](const ngraph::Node::type_info_t & typeInfo) {
-        return std::string(typeInfo.name) + "/" + std::to_string(typeInfo.version);
-    };
-
     std::ostringstream err_log;
 
-    std::queue<std::pair<std::shared_ptr<ngraph::Node>, std::shared_ptr<ngraph::Node>>> q;
+    using ComparedNodes = std::pair<std::shared_ptr<ngraph::Node>, std::shared_ptr<ngraph::Node>>;
+    std::queue<ComparedNodes> q;
```

Comment:
What is the purpose of ComparedNodes if it is used only in one place?

Reply:
To get meaningful name for generic type like `std::pair`.